### PR TITLE
Add files for EE

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Try to make this collection EE ready, with tests.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
collection

### CODE REVIEW

File: .github/workflows/ee.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+          python -c
+          'import yaml ;
+          f = open("galaxy.yml", "rb") ;
+          data = yaml.safe_load(f) ;
+          f.close() ;
+          data["version"] = data.get("version") or "0.0.1" ;
+          f = open("galaxy.yml", "wb") ;
+          f.write(yaml.dump(data).encode("utf-8")) ;
+          f.close() ;
+          '
```
Comment (briantist) [inline .github/workflows/ee.yml:58]: why is this needed? does your `galaxy.yml` usually not have a version?

**Review by briantist — APPROVED** [review]
one question but otherwise looks good, I do wonder about being able to just re-use existing integration tests maybe, instead of having to write dedicated ones 🤔 I suppose this might vary a lot between collections.

File: .github/workflows/ee.yml (hunk 1 above)
Comment (felixfontein) [reply .github/workflows/ee.yml:58]: This isn't needed for this collection, but I intend this workflow (similar to the import-galaxy one) to be copy'n'paste-able in other collections which might not have a version in galaxy.yml.

Comment (felixfontein) [reply .github/workflows/ee.yml:58]: (This should probably be turned into a reusable workflow, I know ... :) )

**Comment by felixfontein** [comment]
About re-using existing integration tests: I thought about this, but then decided against it, since integration tests usually set up their environment, while here we want to test whether the generated EE contains everything we need. Which is different enough that it warrants having its own set of tests.

I guess it will be possible to somehow combine them, or at least parts of them, but for a first approximation I thought having two different sets of tests is probably a better start. (Even though right now they are pretty much identical...)

**Comment by felixfontein** [comment]
@briantist thanks for reviewing this!
